portfolio/src/main/webapp/script.js

```diff
@@ -12,12 +12,19 @@
 // limitations under the License.
 
 
-fetch('/data').then(response => response.json()).then((comment) => {
-	let doc = document.getElementById('comment-container');
-	for(x in comment){
-		doc.innerHTML += "<li>";
-		doc.innerHTML += "<h3>" + comment[x].name + "</h3>";
-		doc.innerHTML += "<p>" + comment[x].content + "</p>";
-		doc.innerHTML += "</li>";
+fetch('/data').then(response => response.json()).then((comments) => {
+	let doc = document.getElementById('comments-container');
+    let ul = document.createElement('ul');
+    doc.appendChild(ul);
+    
```

Comment:
Looks like we've got an intermixing of tabs and spaces here, hence the odd indenting. You can often configure your editor to insert spaces when you hit the Tab key, which makes this sort of thing less of an issue.